zaas-service/src/test/java/org/zowe/apiml/zaas/security/config/AttlsConfigTest.java

```diff
@@ -43,26 +45,33 @@
         "server.service.scheme=http"
     }
 )
-@ActiveProfiles({"acceptance", "AttlsConfigTest"})
 @TestInstance(Lifecycle.PER_CLASS)
```

Comment:
I think it's good to have a spring profile marked for context tests, to isolate them if need to, could we keep the `@ActiveProfiles("AttlsConfigTest")`?

Reply:
Actually it is used nowhere, I have removed couple of test configuration beans, because they are not needed anymore